_posts/2016-02-18-starter-issues.md

```diff
@@ -0,0 +1,112 @@
+
+```
+- [ ] Start a Pull Request. Set description to `closes #123`
+If this is your first, welcome :tada: :smile: [Here is a great tutorial](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github) on how to send a pull request)
+```
+
+Once the PR is created, we check all above boxes and replace the above item with something like `- [x] Start a Pull Request (#123)`, where `#123` is the number of the Pull Request.
+
+If this issue is about code, a test needs to be written or adapted. We give guidance where the test needs to be added, ideally like to the files / lines
+
+```
+- [ ] Create a file `/test/unti/my-method-test.js` in [/test/unit](https://github.com/myuser/mylib/tree/master/test/unit). See [/test/unit/my-other-method-test.js](https://github.com/myuser/mylib/tree/master/test/unit/other-method-test.js) for comparison
+- [ ] Add a check that `myLib.myMethod` is a function in [/test/integration/constructor-test.js](https://github.com/myuser/mylib/tree/master/test/integration/constructor-test.js)
+```
+
+Then we ask the user to commit and push the tests. This way the commit will trigger our continuous integration tests on [Travis CI](https://travis-ci.org) and will add a red **×** to the commit, so we can be sure that tests are failing as expected. Here’s [an example](https://github.com/hoodiehq/pouchdb-hoodie-api/pull/100/commits) of what I mean.
+
+```
+- [ ] Make sure that tests fail when run with `npm test`. Commit with `test: myLib.myMethod` and push
+```
+
+Now comes the fun, the “actual” work, and all it takes is an item describing it. If helpful, we link to the right place and explain how the feature / bug fix can be implemented. But it’s okay to keep it short and leave something for the contributor to figure out. It’s fun after all, and at this point the contributor should feel comfortable to ask for help if they get stuck. So often time we simply do:
+
+```
+- [ ] Implement :)
+- [ ] Make sure tests pass when run with `npm test`. Commit with `feat: myLib.myMethod` and push
+```
+
+The work is done. Now someone else needs to review and merge the work.
+
+```
+- [ ] Someone else: review & merge
+```
+
+During the review it often turns out that something is wrong or missing, so the pull request needs amendments. The contributor can simply add more commits as long as needed. Once they are done, they can either squash the commits themselves if they know how to do it, or the reviewer does it for them. Very important when the reviewer is cleaning up the commit messages is to keep the original author. Nothing is more disappointing than a first pull request accidentally “stolen” by a reviewer. A custom author can be set with `git commit --author="Pat Doe <pat@example.com>""`
```

Comment:
I suggest:

"During the review it may turn out something is not quite right, so the pull request needs amendments"

As 1) "wrong" sounds harsh. 2) It doesn't always happen.